### DISCUSSION TITLE
Pages not implementing CSS styles

### DISCUSSION THREAD

**Discussion by itsleaheye** [community/community#22722] [Other Feature Feedback, Questions, & Ideas]
<p>Link to my repository: <a href="https://github.com/leahnonay/Portfolio" rel="nofollow noopener">https://github.com/leahnonay/Portfolio</a><br>
Link to the live portfolio: <a href="https://leahnonay.github.io/Portfolio/" rel="nofollow noopener">https://leahnonay.github.io/Portfolio/</a></p>
<p>I cannot figure out why my website is not implementing any CSS styles. I saw some posts suggesting to change the pathing but I tried that already and both: ‘/css/styles.scss’ and ‘css/styles.scss’ are not returning any style sets. The site works flawlessly otherwise (Tested it on multiple platforms), it just does not seem to work when published through GitHub Pages. Any ideas how to fix this?</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/leahnonay">@leahnonay</a>, welcome to the GitHub Support Community!</p>
<p>From what I can tell, I believe it’s the extension of your main CSS stylesheet. Changing it from styles.<strong>scss</strong> to styles.<strong>css</strong> appears to fix the display issue.</p>

**Comment by itsleaheye** [comment]
<p>I modified it to those changes you suggested but I do not think the file updates are affecting the live version. I checked the live link and it still is trying to pull from ‘.scss’ instead of how the file says ‘.css’</p>

**Comment by SethClydesdale** [comment]
<p>It may take a bit for changes to take effect. Most commonly you’ll need to clear your cookies/cache if the old issue is still persisting.</p>
<p>On my end it appears to be fine now. :+1:</p>
